nativelib/src/main/resources/scala-native/zone/LargeMemoryPool.c

```diff
@@ -0,0 +1,86 @@
+#include <stdio.h>
```

Comment:
In theory, most of these C sources could be implemented in Scala Native instead of C, I think we should be able to have binding to all the needed headers. However, I don't require you to rewrite it. 
I guess that one of the reasons to implement them in C is to make them a bit faster and independent from GC and it's allocation. Is there maybe some other reason?

Reply:
At the beginning I simply wanted to make the zone backend as fast as possible and be maintained the same way as GC does. Hmm, implementing it in Scala Native should be better for maintenance, I'll consider rewriting it later.